users.md

```diff
@@ -40,6 +40,10 @@ title: Users
         <div class="col-md-9 col-s-12">
           <h2>{{user.name}}</h2>
           <p>{{user.testimonial}}</p>
+          {% if user.url == nil %}
+          {% else %}
+          <p><a href="{{user.url}}" target="_blank">View {{user.name}}'s site</a></p>
```

Comment:
We might want another parameter like linktext or so and use that .. it could different for each one.

E,g. Find out more on the Line engineering blog

Reply:
Thats done now